src/main/java/racingCar/model/Car.java

```diff
@@ -0,0 +1,24 @@
+package racingCar.model;
+
+public class Car {
+    private static final int CAN_MOVE_NUMBER = 4;
+
+    private int location = 0;
+
+    public boolean canMove(int randomNumber) {
+        return randomNumber >= CAN_MOVE_NUMBER;
+    }
+
+    public int move(int randomNumber){
```

Comment:
```suggestion
    public int move(int randomNumber) {
```

컨벤션을 위반하셨습니다!
IntelliJ Auto Formatting 기능을 사용하는 습관을 들여보아요 :)

The auto-formatting shortcut in IntelliJ

* For windows Ctrl + Alt + L.
* For ubuntu Ctrl + Alt + windows + L.
* For Mac Command + Option + L.